src/pug/blog/report_25.pug

```diff
@@ -0,0 +1,98 @@
+extends _blog.pug
+
+block append subtitle
+  | Progress report #25
+
+block content
+  h2 Progress report #25
+  +news_header('13 September 2024', 'report_24', '')
+
+  h3 Wireless RF prototypes 6 and 7
+  p We continued optimizing antenna performance by doing #[em "trim tuning"], adding a long copper stub on the antenna's end, and then taking measurements while progressive scratching off the copper and reducing the stub length.
+
+  p In the #[em v6] iteration the antenna was placed horizontally, but unfortunately the antenna was detuned so much that the center frequency was under the target no matter the stub length.
+
+  p In the #[em v7] iteration we were able to scratch the stub (~1.25 mm under the recommended length) to the target center frequency of #[em 2445 MHz], these measurements were performed with a #[em VNA] (Vector Network Analyzer) device by #[em Fabio], and later replicated inhouse at Input Labs lab.
+
+  +figure(
+    '',
+    'RF test 6',
+    '/static/blog/RF_proto_6.jpg',
+    'small inline',
+  )
+  +figure(
+    '',
+    'RF test 7 (unmodified)',
+    '/static/blog/RF_proto_7.jpg',
+    'small inline',
+  )
+  +figure(
+    '',
+    'RF test 7 (tuned)',
+    '/static/blog/RF_proto_7b.jpg',
+    'small inline',
+  )
+
+  p The modified v7 prototype has substantially better antenna performance, but it is still far from reaching its full potential, reason being that with the "trim tuning" method we can only tune the antenna itself but not the inductance/capacitance of the matching network. To actually tune the matching network we would need more advanced tools and expertize we don't have, or outsource the tuning to an external company (expensive).
+
+  p We also found that by reducing the rate of the #[em ESB] (Enhanced Shock Burst) connection from #[em 2 Mbps] to #[em 250 Kbps] (8x times slower) the reception sensitivity increases by several decibels. #[i (see table below)]
```

Comment:
This is not correct. The sensitivity remains the same. However, at a reduce throughput, you have fewer lost packets at the same sensitivity.
Below is not a table, but two bullet points